src/net/tcp/stream.rs

```diff
@@ -88,6 +90,8 @@ impl TcpStream {
         let socket = new_for_addr(addr)?;
         #[cfg(unix)]
         let stream = unsafe { TcpStream::from_raw_fd(socket) };
+        #[cfg(target_os = "hermit")]
+        let stream = unsafe { TcpStream::from_raw_fd(socket) };
```

Comment:
Nit: I think we can merge this with the Unix code as it's the same API, even though it's two different traits.

Reply:
Yeah, I oversaw it